bangazonapi/models/store.py

```diff
@@ -5,10 +5,8 @@
 
 
 class Store(models.Model):
+    name = models.CharField(max_length=255)
     description = models.CharField(max_length=255)
```

Comment:
### Recommendation: 
Change description to models.TextField(max_length=2500)
### Reason: 
So that the store owner can provide a more verbose description of their store and are not limited to 255ch